corehq/apps/userreports/models.py

```diff
@@ -164,7 +158,6 @@ class DataSourceConfiguration(UnicodeMixIn, CachedCouchDocumentMixin, Document,
     """
     domain = StringProperty(required=True)
     engine_id = StringProperty(default=UCR_ENGINE_ID)
-    es_index_settings = SchemaProperty(ElasticSearchIndexSettings)
```

Comment:
Are these schema changes done for pre-existing documents as well? i.e it would be removed from the existing couch documents just like dropping a column from SQL. I assume no.

Reply:
they are not proactively removed, but this feature wasn't used by any real projects anyways, so it shouldn't be an issue

Reply:
Cool, thanks